webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/AgglomerateService.scala

```diff
@@ -197,8 +198,8 @@ class AgglomerateService @Inject()(config: DataStoreConfig) extends DataConverte
         Tree(
           treeId = math.abs(agglomerateId.toInt), // used only to deterministically select tree color
           createdTimestamp = System.currentTimeMillis(),
-          nodes = nodes,
-          edges = skeletonEdges,
+          nodes = ArraySeq.unsafeWrapArray(nodes),
```

Comment:
Why unsafe?

Reply:
The old code implicity called a conversion, which meant a full copy of the thing. This implicit conversion is now deprecated, meaning I had to either add an explicit conversion, or, as it suggested, do this unsafeWrapArray *IF* I know that the underlying array is never mutated. This way, the copy is gone. It’s called unsafe because if the underlying array is mutated, the scala-guaranteed immutability of `Seq` is broken. But we don’t mutate anything in this context, so I think this is the best option

Reply:
Ok. Maybe document why unsafe is fine with a comment?